e2e-tests/src/cases.rs

```diff
@@ -25,6 +26,7 @@ pub type PossibleTestCases = Vec<(&'static str, TestCase)>;
 /// This comes up in local tests.
 pub fn possible_test_cases() -> PossibleTestCases {
     vec![
+        ("version_upgrade", test_schedule_version_change),
```

Comment:
`finalization` testcase should remain first - if finalization is gone, nothing else matters